dev/io.openliberty.microprofile.openapi.ui.internal_fat/fat/src/io/openliberty/microprofile/openapi/ui/internal/fat/tests/FATSuite.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.openapi.ui.internal.fat.tests;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.containers.TestContainerSuite;
+
+@RunWith(Suite.class)
```

Comment:
Should this have `@MinimumJavaLevel(javaLevel = 11)`?

Reply:
Going by the documentation at https://github.com/OpenLiberty/open-liberty/wiki/FAT-Tests#skipping-tests-based-on-java-level this isn't necessary when using `fat.minimum.java.level` in the bnd.bnd.